addons/sale_project/models/project.py

```diff
@@ -173,6 +173,17 @@ def action_open_project_invoices(self):
             action['res_id'] = invoice_ids[0]
         return action
 
+    #todo retravailler l'action[views] atm makes no sense.
+    def action_aal(self, section_name, domain=None, res_id=False):
+        self.ensure_one()
+        if section_name in ['other_costs', 'other_revenues']:
```

Comment:
`else`, `return action` is going to crash, isn't it?

Reply:
it doesn't. But the if section name is actually useless, so i can just remove it and return the action in every case.